### PR TITLE
Miscellaneous BCC patches

### PR DESCRIPTION
**Pull request by joelagnel**
Here are 3 miscellaneous patches while I prepare the broader set of changes for making BCC work with BPFd.

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: src/python/bcc/table.py [hunk 1]
```diff
@@ -430,11 +430,11 @@ def __setitem__(self, key, leaf):
 
     def __delitem__(self, key):
         key = self._normalize_key(key)
-        # Deleting from array type maps does not have an effect, so
-        # zero out the entry instead.
+        super(ArrayBase, self).__delitem__(key)
+
+    def clearitem(self, key):
         leaf = self.Leaf()
```
Comment (yonghong-song) [inline src/python/bcc/table.py]: should we do _normalize_key here?

Comment (joelagnel) [reply src/python/bcc/table.py]: Yes, The user's of `ArrayBase`'s __delitem__ after this patch are, `PerfEventArray` and `ProgArray`. Both these were calling `_normalize_key` in the previous code (before this patch). 

However I screwed up the normalization for __delitem__ in `PerCpuArray` and `Array`. Those need to normalize too before calling `clearitem`. Or `clear_item` itself should normalize within. I'll fix that in the next rev. thanks

File: src/python/bcc/table.py [hunk 2]
```diff
@@ -665,6 +661,10 @@ def __getitem__(self, key):
     def __setitem__(self, key, leaf):
         super(PerCpuArray, self).__setitem__(key, leaf)
 
+    def __delitem__(self, key):
+        # Delete in this type does not have an effect, so zero out instead
```
Comment (joelagnel) [inline src/python/bcc/table.py]: Need to add a `_normalize_key` here, on in `clearitem`

File: src/python/bcc/table.py (hunk 1 above)
Comment (joelagnel) [reply src/python/bcc/table.py]: Fixed this.

**Pull request merged**